nbconvert/preprocessors/execute.py

```diff
@@ -184,7 +184,7 @@ def run_cell(self, cell):
         while True:
             try:
                 timeout = self.timeout
-                if timeout <= 0:
```

Comment:
> If 0 does already work to suspend the timeout, that's fine

It does not,  it times out immediately so the `<=` introduce by previous PR did actually change the behavior. 
I'm just reverting what it did.

I personally think that `0` , `-1`, and `None` should have the same effect, but that the opposite of what was decide in previosu PR.